index.html

```diff
@@ -29,11 +29,11 @@
   <section>
       <div class="introduction">
         <div class="social-Dskt">
-        <img src="images/github.png" alt="github-icon">
-        <img src="images/in.png" alt="linked-in">
-        <img src="images/angelist.png" alt="angelist">
-        <img src="images/newtwitter.png" alt="twitter-icon">
-        <img src="images/media.png" alt="Media">
+        <a href=""><img src="images/github.png" alt="github-icon"></a>
+        <a href=""><img src="images/in.png" alt="linked-in"></a>
+        <a href=""><img src="images/angelist.png" alt="angelist"></a>
+        <a href=""><img src="images/newtwitter.png" alt="twitter-icon"></a>
+        <a href=""><img src="images/media.png" alt="Medium"></a>
```

Comment:
[OPTIONAL] Since these social-media icons are also in a form of list, you can apply the same `ul>li`  here to:
![image](https://user-images.githubusercontent.com/61361037/229898870-035d38f4-9d25-45a0-b072-a8ba8d6ffc36.png)

The purpose is to improve the browser accessibility, so that browser could consider it as a `list` of social media icons.